kubectl-plugin/pkg/cmd/get/get_cluster.go

```diff
@@ -68,7 +69,10 @@ func (options *GetClusterOptions) Complete(args []string) error {
 		options.AllNamespaces = true
 	}
 
-	options.args = args
+	if len(args) >= 1 {
```

Comment:
there should never be more than one positional argument, but just ignore the rest instead of erroring if there is for some reason